pkg/snowflake/database_test.go

```diff
@@ -54,3 +56,14 @@ func TestDatabaseCreateFromDatabase(t *testing.T) {
 	q := db.Create()
 	r.Equal(`CREATE DATABASE "db1" CLONE "abc123"`, q)
 }
+
+func TestListDatabases(t *testing.T) {
+	r := require.New(t)
+	mockDB, mock, err := sqlmock.New()
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
ineffectual assignment to `err` (ineffassign)